snappy_pipeline/workflows/panel_of_normals/__init__.py

```diff
@@ -227,6 +249,40 @@ class Mutect2StepPart(PanelOfNormalsStepPart):
         ),
     }
 
+    #: Class key/values extensions
```

Comment:
Do you need this? I'm eyeballing it, but it seems like the extensions are the same for both actions...

Reply:
That's correct, but it's by chance, there is no need for it to be the case. So I thought I would keep them separate, but I don't know if it's good practice.

Reply:
Maybe it can be simplified as you don't even have different methods for the different actions.
```python
    @dictify
    def get_output_files(self, action):
        """Return panel of normal files"""
        self._validate_action(action)
        ext_dict = {
            "vcf": "vcf.gz",
            "vcf_md5": "vcf.gz.md5",
            "tbi": "vcf.gz.tbi",
            "tbi_md5": "vcf.gz.tbi.md5",
        }
        tpls = {
            "prepare_panel": "work/{{mapper}}.{tool}.prepare_panel/out/{{normal_library}}.{ext}",
            "create_panel": "work/{{mapper}}.{tool}.create_panel/out/{{mapper}}.{tool}.panel_of_normals.{ext}",
        }
        for key, ext in ext_dict.items():
            yield key, tpls[action].format(tool=self.name, ext=ext)

    @dictify
    def get_log_file(self, action):
        """Return panel of normal files"""
        self._validate_action(action)
       ext_dict = {
            "conda_list": "conda_list.txt",
            "conda_list_md5": "conda_list.txt.md5",
            "conda_info": "conda_info.txt",
            "conda_info_md5": "conda_info.txt.md5",
            "log": "log",
            "log_md5": "log.md5",
        }
        tpls = {
            "prepare_panel": "work/{{mapper}}.{tool}.prepare_panel/log/{{normal_library}}.{ext}",
            "create_panel": "work/{{mapper}}.{tool}.create_panel/log/{{mapper}}.{tool}.panel_of_normals.{ext}",
        }
        for key, ext in ext_dict.items():
            yield key, tpls[action].format(tool=self.name, ext=ext)
```